ethereum/nft/trades/trades.sql

```diff
@@ -18,6 +21,12 @@ CREATE TABLE nft.trades (
     exchange_contract_address bytea NOT NULL,
     tx_hash bytea NOT NULL,
     block_number integer,
+    nft_token_ids_array text[],
```

Comment:
Maybe just briefly explain why these arrays exist. I.e "Sometimes many trades occur in a given trade" etc.

Reply:
Thank you so much @va3093. Addressed your comments in a new commit.